client/src/views/Game.vue

```diff
@@ -60,13 +87,12 @@
         <!-- Current cards in the deck -->
         <v-col class="mb-6" v-if="gameState.status === 'Playing'">
           <v-card
-            v-if="gameState.status !== 'Finished'"
+            v-if="gameState.current_player != undefined &&  gameState.player_id === gameState.current_player.id"
```

Comment:
Does `current_player` become undefined / empty when the game is finished?

Reply:
Before the game starts it does. I'm not sure when it's finished, but I imagine it would eventually.

Reply:
You know... I did not actually finish a full game to see what that button does. I'll check now.

Reply:
So. I've noticed the game does not have a winning condition finished right now..... so it keeps playing forever and ever, beacuse the backend doesn't report the game is `finished`.... However, that whole Section I have wanted to update a bit. I put in some updates there now.

Reply:
There is a group working on the winning condition. Hopefully they finish their work soon.